### DISCUSSION TITLE
【思路】可以尝试用firefox-docker登录账号解除风控

### DISCUSSION THREAD

**Discussion by wei-z-git** [0-8-4/miui-auto-tasks#256] [展示和讲述 ｜Show and tell]
可以在服务器启动一个docker-firefox容器，然后用它去访问account.xiaomi.com, 这样相当于使用了同一ip去登录，比重新创建一个windows VM更简单
项目地址：
https://github.com/jlesage/docker-firefox

**Comment by wei-z-git** [comment]
好吧，原来已经有人想到这个方法了。。😂

**Reply by 0-8-4** [reply]
问题不大

**Comment by superHao2000** [comment]
可行吗
